src/plugins/Dropbox/index.js

```diff
@@ -76,48 +76,40 @@ module.exports = class Dropbox extends Plugin {
   }
 
   isFolder (item) {
-    return item.is_dir
+    return item['.tag'] === 'folder'
   }
 
   getItemData (item) {
-    return Object.assign({}, item, {size: item.bytes})
+    return item
   }
 
   getItemIcon (item) {
-    var icon = icons[item.icon]
-
-    if (!icon) {
-      if (item.icon.startsWith('folder')) {
-        icon = icons['folder']
-      } else {
-        icon = icons['page_white']
-      }
-    }
-    return icon()
+    return icons[item['.tag']]()
   }
 
   getItemSubList (item) {
-    return item.contents
+    return item.entries
   }
 
   getItemName (item) {
-    return item.path.length > 1 ? item.path.substring(1) : item.path
+    return item.name || ''
```

Comment:
Is this ever empty in the API?

Reply:
only in the case of the root folder, where a name is not added.